dg/commands/ibmcloud/oc/cluster/create.py

```diff
@@ -184,19 +188,51 @@ def create(
         },
     )
 
+    # without this step, "oc login …" fails with the following error:
+    # Error from server (InternalError): Internal error occurred: unexpected response: 500”
+    execute_ibmcloud_command(["oc", "cluster", "config", "--cluster", cluster_name])
+
     if workaround:
         # workaround START - TODO remove when "no such host" issue is fixed
         # (see IBM Cloud support case CS2206770)
-        logging.info("Restarting openshift-dns pods")
+        logging.info("Waiting for the Image Registry hostname to be known")
         cluster = dg.config.cluster_credentials_manager.cluster_credentials_manager.get_cluster(server)
```

Comment:
does the log message and the line below really match? When having a look at `get_cluster` it is just mentions that the method returns the meta data for the cluster.

Reply:
This log statement corresponds to the whole workaround. I added a blank line below it to make it more clear.